src/Microsoft.AspNet.Localization/RequestLocalizationMiddleware.cs

```diff
@@ -62,6 +74,10 @@ public RequestLocalizationMiddleware(RequestDelegate next, RequestLocalizationOp
             {
                 foreach (var provider in _options.RequestCultureProviders)
                 {
+                    if (provider is RequestCultureProvider)
+                    {
+                        ((RequestCultureProvider)provider).Options = _options;
```

Comment:
Hmm, downcasts always look suspicious. Is there any design issue here that needs to be addressed?

cc @DamianEdwards